specification/src/chapter2.adoc

```diff
@@ -506,7 +591,7 @@ examples later in this specification.
 
 | CAT_NNN
 | A security platform attestation MUST be signed by a HW RoT, if present, or by
-trusted security services
+a FW RoT
```

Comment:
```suggestion
a FW RoT (which has in its direct TCB a HW RoT)
```

Reply:
See comment above about HW RoT. We can't assume that every RISC-V system will have one.

Reply:
how about adding - "a FW RoT (bootstrapped by a HW RoT if available on the system)"